apis/v1alpha1/sealedsecret.go

```diff
@@ -76,9 +80,13 @@ func (sl *SealedSecretList) GetListMeta() metav1.List {
 	return &sl.Metadata
 }
 
-func labelFor(o metav1.Object) []byte {
-	label := fmt.Sprintf("%s/%s", o.GetNamespace(), o.GetName())
-	return []byte(label)
+func labelFor(o metav1.Object) ([]byte, bool) {
+	label, ok := o.GetAnnotations()[SealedSecretLabelAnnotation]
+	if ok && label != "" {
```

Comment:
May as well drop the `ok` here, since we only want to know if it is a non-empty value:

```golang
label := o.GetAnnotations()[SealedSecretLabelAnnotation]
if label != "" {
  ...
}
```